.github/workflows/test-with-plugins.yml

```
@@ -1,10 +1,8 @@
-name: foreman-js ci
-on:
-  pull_request:
-    branches: [master]
+name: Test foreman plugins with foreman-js
+on: [push, pull_request, workflow_dispatch]
```

```suggestion
on: [push, pull_request, workflow_dispatch]
```
why do we want to run it on all of these events?
keep in mind, we have limit of workers for organization, we should run only jobs we consider important.
Why would we trigger the tests manually? Are we interested in Pull request to different branches?

---

> Are we interested in Pull request to different branches?

Yes, see my comment about the `next` branch and the `next/**` pattern.

> Why would we trigger the tests manually?

Isn't it nice to have?
As a maintainer, I want some sort of freedom to push changes to a branch that is not `master` so I can run tests in the cloud.
I was wondering what would be better, have a button in Github that will run tests against my branch, or have it automated when I push. I tend to prefer the automated way because we have more than one button to click (one for each workflow).